docroot/.htaccess

```diff
@@ -125,8 +125,10 @@ AddType application/octet-stream .rte
   RewriteCond %{ENV:IS_DDEV_PROJECT} !^$ [NC]
   RewriteCond %{HTTPS} off
   RewriteCond %{HTTP:X-Forwarded-Proto} !https
+  RewriteCond %{HTTP_HOST} ^mass\.local$
```

Comment:
@clairesunstudio Is there a reason for this rule being in place here?

Reply:
This was added by @arthurbaghdas. Arthur, can you confirm that this is intentional?

Reply:
Yes, this is to isolate this behavior. I was having issues running the phpunit and behat tests locally, because it was trying to redirect http://mass-web to https://mass-web

Reply:
Removed the code, seems like it was only happening for my setup